xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -314,7 +340,7 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the name of the server the client is trying to connect to. That name is used for server certificate validation. It can be a DNS name or IP address.</summary>
```

Comment:
```suggestion
        <summary>Gets or sets the name of the server the client is trying to connect to. That name is used for server certificate validation. It can be a DNS name or an IP address.</summary>
```